docs/src/rsh/consensus/index.md

```diff
@@ -370,15 +370,15 @@ where `{!rsh} PAY_AMOUNT_EXPR` is an expression that evaluates to a [pay amount]
 performs a transfer of network tokens or non-network tokens from the contract to the named participant.
 The amount transfered must evaluate to less than or equal to the balance of the network and non-network tokens in the contract account.
 
-A transfer expression may only occur within a consensus step.
+A `{!rsh} transfer` expression may only occur within a consensus step.
```

Comment:
I don't think `transfer expression` should be monospaced, because the first mention of it is in standard format. (See paragraph above this one)
IDK why. Perhaps @jeapostrophe could educate us on the nuance of not monospacing "transfer expression"?

Reply:
The subject of this sentence is the difference between using a word in its English meaning and talking about a particular word, like "subject", itself.

There is a English meaning to "transfer" and when we're writing a sentence like "Alice did a transfer of funds to the contract", we're not talking about the Reach function `transfer`, but we are talking about that when we say, "This block of code shows an example of using `transfer` to transfer funds."